File: src/main/java/pl/mleczkomatyaszek/SoundShare/Model/UserModel.java
package pl.mleczkomatyaszek.SoundShare.Model;

import lombok.Data;
import lombok.RequiredArgsConstructor;

@Data
@RequiredArgsConstructor
public class UserModel {

    private String username;
    
    private String password;

    private String firstName;

    private String lastName;

    private String email;

}

File: src/main/resources/frontend/SoundShareApp/src/app/models/entities/user/user.model.ts
import { Role } from './../role/role.model';
export class User{
    
    public user_id: number;
    public username: string;
    public password: string;
    public firstName: string;
    public lastName: string;
    public email: string;

    constructor(user_id: number, username: string, password: string, firstName: string, lastName: string, email: string) {
        this.username = username;
        this.password = password;
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
    }

}